src/org/sosy_lab/java_smt/solvers/mathsat5/Mathsat5NativeApiTest.java

```diff
@@ -79,6 +85,7 @@ public void createEnvironment() {
     long cfg = msat_create_config();
 
     msat_set_option_checked(cfg, "model_generation", "true");
+    msat_set_option_checked(cfg, "proof_generation", "true");
```

Comment:
Why enable proofs for all tests and then build a new, dedicated solver environment for the proof test?